### PR TITLE
engine: add known issues to 20.10.15 release notes

### PR DESCRIPTION
**Pull request by thaJeztah**
- relates to https://github.com/docker/cli/issues/3595
- relates to https://github.com/docker/cli/pull/3592

With this PR:

Screenshot 2022-05-10 at 11 15 55

### CODE REVIEW

**Comment by thaJeztah** [comment]
@usha-mandya @dockertopia ptal

File: engine/release-notes/index.md [hunk 1]
```diff
@@ -29,6 +29,15 @@ for Docker Engine.
 This release of Docker Engine comes with updated versions of the `compose`,
 `buildx`, `containerd`, and `runc` components, as well as some minor bugfixes.
```
Comment (dockertopia) [inline engine/release-notes/index.md:30]:
```suggestion
`buildx`, `containerd`, and `runc` components, as well as some minor bug fixes.
```

**Review by dockertopia — COMMENTED** [review]
``

File: engine/release-notes/index.md [hunk 2]
```diff
@@ -29,6 +29,15 @@ for Docker Engine.
...
 
+> **Known issues**
+> 
+> We identified an issue with the [macOS CLI binaries](https://download.docker.com/mac/static/stable/)
+> in the 20.10.15 release. A fix for this issue [has been merged](https://github.com/docker/cli/pull/3592)
```
[the remark this answers is not in the document]
Comment (thaJeztah) [reply engine/release-notes/index.md:35]: Ah, yes, I should do those (I know we haven't been doing for this document, but it's indeed the "correct" way to do it)

[the remark this answers is not in the document]
Comment (thaJeztah) [reply engine/release-notes/index.md:35]: As to changing `fix for this issue has been merged` to `fix`; I generally try to make the title of the link slightly longer to help discoverability, and to make it easier to use (`fix` is very short for a link, and especially on mobile devices may be more tricky to "point" at)

**Comment by thaJeztah** [comment]
Updated with your suggestions; changed the `fix` for the link to use `fix for this issue` (instead of the "has been merged"); PTAL if this looks good to you 🤗 

Screenshot 2022-05-10 at 13 08 08

**Comment by usha-mandya** [comment]
Thanks @thaJeztah. Looks good. There's a broken link to the Extensions doc which isnt part of this PR. I've re-run the failed jobs to see if that fixes any issues.

**Comment by thaJeztah** [comment]
Bleh.. looks to be still failing; broken link may be in the external repository though;

```
- ./_site/desktop/extensions-sdk/extensions/labels/index.html
Ran on 1666 files!


  *  linking to internal hash #design-principles that does not exist (line 197)
     <a href="../../design/design-overview/#design-principles">CSS guidlines</a>
```

**Comment by thaJeztah** [comment]
Fix for the above is in https://github.com/docker/extensions-sdk/pull/160

**Review by usha-mandya — APPROVED** [review]
LGTM

**Pull request merged**